cartofacade/capi_test.go

```diff
@@ -138,13 +138,13 @@ func TestCGoAPI(t *testing.T) {
 		pc, err := pointcloud.ReadPCD(file)
 		test.That(t, err, test.ShouldBeNil)
 
-		// test invalid addSensorReading: valid reading binary
+		// test valid addSensorReading: valid reading binary
```

Comment:
Was this a typo? There are no errors from these calls

Reply:
yes it was a typo